Resources/Prototypes/Recipes/Reactions/chemicals.yml

```diff
@@ -81,6 +81,7 @@
 - type: reaction
   id: PotassiumExplosion
   impact: High
+  priority: 2
```

Comment:
better to make these `20`, `10` etc rather than `2` or `1` so we can have some granularity without changing a bunch of other shit